src/shared/components/query/GeneticProfileSelector.tsx

```diff
@@ -150,7 +150,7 @@ export default class GeneticProfileSelector extends QueryStoreComponent<{}, {}>
 		{
 			output.push(
 				<div key={output.length} className={styles.zScore}>
-					Enter a z-score threshold ±:
+					Enter a z-score threshold &plusmn;:

```

Comment:
There are two corruption issues here.  THe close button in dialog and the plusmn.  The close button must be fixed with css hack b/c sp char is in bootstrap files.  I will do that as I've fixed it elsewhere.   The plusmn, let me just verify that entity doesn't fix it.  I would shocked if jitpack could really corrupt an HTML entity